src/common/metrics/admissions_per_100k.tsx

```diff
@@ -22,24 +22,28 @@ export const AdmissionsPer100kMetric: MetricDefinition = {
   metricNameForSummaryStat: 'Weekly New Admissions',
 };
 
+const LIMIT_LOW = 10;
+const LIMIT_MEDIUM = 19.9;
```

Comment:
Maybe make `LIMIT_MEDIUM = 19.999` since the High is when _weekly new admissions_ >= 20.0